### ISSUE TITLE
Blocking JS: Inline dnthelper

### ISSUE THREAD

**Issue by atopal** [mdn/sprints#257]
Taks
- [ ] inline dnt helper script
- [ ] measure and record performance impact

Acceptance Criteria:
- [ ] no external request to dnthelper file
- [ ] performance impact reported

**Comment by jwhitlock** [comment]
@atopal I am surprised that measuring the effect on performance is not part of the acceptance criteria.

**Comment by atopal** [comment]
Yeah, that's a good point. That should be part of all the acceptance criteria

**Comment by atopal** [comment]
Apparently work has already started in https://github.com/mozilla/kuma/pull/4881

**Comment by jwhitlock** [comment]
With the ZenHub plugin, I can connect pull requests in mozilla/kuma with issues in mdn/sprints. They aren't visible if you don't have the plugin, though.

**Comment by jwhitlock** [comment]
Duplicate of https://github.com/mdn/mdn-fiori/issues/31

**Comment by jwhitlock** [comment]
Actually, going to dupe the other way.

**Comment by jwhitlock** [comment]
Well, whichever way this is tracked, it went to production today.

**Comment by schalkneethling** [comment]
@atopal @jwhitlock Had a look at Speedcurve for the last seven days. We can clearly see where the change happened. In the below screenshot there are 5 JS requests:

<img width="1336" alt="screen shot 2018-07-31 at 13 29 55" src="https://user-images.githubusercontent.com/10350960/43457083-07e22704-94c6-11e8-8ee7-362e8b2db064.png">

Later the same day, after the deployment, it went down to 4 requests:

<img width="1323" alt="screen shot 2018-07-31 at 13 30 06" src="https://user-images.githubusercontent.com/10350960/43457101-1b6c27ac-94c6-11e8-9346-8ede3ffb387b.png">

As a result the time to DOM content loaded went from ~2.32s

<img width="1350" alt="screen shot 2018-07-31 at 13 30 19" src="https://user-images.githubusercontent.com/10350960/43457150-3d835054-94c6-11e8-8da2-9c4c71a1b4ba.png">

To ~2.01s

<img width="1337" alt="screen shot 2018-07-31 at 13 30 28" src="https://user-images.githubusercontent.com/10350960/43457173-4df50d92-94c6-11e8-9f50-a9f0e0c04665.png">

We are thus saving ~300ms from this change.

**Comment by atopal** [comment]
@schalkneethling Time to DOM content load is not part of a primary or secondary performance KPI for MDN. Can you please test with one of those?

Our main performance KPI, xx-editor load time, shows [no movement at all for the 90th percentile or the median](https://docs.google.com/spreadsheets/d/1e2_481_W3EZV4CNOp34zqSH2Uc8jJf9ynWkYjtnK_38/edit#gid=1957704471) shortly before or after July 25.

**Comment by schalkneethling** [comment]
@atopal I will have a look. It is of course entirely possible that it is making a very small difference. Perhaps more noticeable in the request/download waterfall.
